proxy-app/utils/yaml-parser.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const yaml = require('js-yaml');
+const path = require('path')
+
+const retrieveApiKey = () => {
+  const pathToFile = path.resolve(__dirname, '../../config.yaml');
+
+  const yamlData = fs.readFileSync(pathToFile, 'utf8');
+
+  try {
+    const constants = yaml.load(yamlData);
+    const apiKey = constants.api_key;
+    if (apiKey) {
+      return apiKey
+    }
+  } catch (error) {
+    if (error) {
+      throw new Error(error.message);
+    }
+  }
+}
+
+module.exports = {
+  retrieveApiKey
+}
```

Comment:
I'm not sure, but this can probably be done in the simplest way. Just store API_KEY in env variables and then use it like `process.env.API_KEY` because in production no one will store api_key in yml file

Reply:
Cool! Thanks for a hint! ) Fixed )